### PR TITLE
test custom runners and autofix

### PR DESCRIPTION
**Pull request by jonapich**
also: fix output by adding PIPE to fix some edge cases of lost stdouts vs errors.

### CODE REVIEW

File: coveo-systools/coveo_systools/subprocess.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def _wrapped_exception(self) -> BaseException:
```
Comment (jonapich) [inline coveo-systools/coveo_systools/subprocess.py:106]: i merged this here by mistake, this was already reviewed in #119

File: coveo-stew/test_coveo_stew/test_stew_ci.py [hunk 2]
```diff
@@ -0,0 +1,357 @@
+import os
+from contextlib import contextmanager
+from typing import Generator, Final
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
```
Comment (klalumiere) [inline coveo-stew/test_coveo_stew/test_stew_ci.py:6]: I had to modified some code recently when I bumped `poetry.lock` because we imported something from `_pytest`. Is there anything else we could use that is less likely to break? Why not the usual `tempfile`? I used it quite a bit in [dump-collector](https://github.com/coveo/dump-collector). :thinking:

**Review by klalumiere — APPROVED** [review]
